assets/styles/layouts/_Header.scss

```diff
@@ -333,6 +333,10 @@
 						color: #ababab !important;
 						font-size: 12px;
 					}
+
+					&::after {
+						height: 1.6em !important;
```

Comment:
All items should have the same height and keep it the same on LA.

<img width="449" alt="Screenshot 2022-06-08 at 09 35 45" src="https://user-images.githubusercontent.com/17085174/172559308-28f3fcf9-2df7-44c1-99d8-053ed991a057.png">